### ISSUE TITLE
Api 21 required for Media package?

### ISSUE THREAD

**Issue by joaquingrech** [jamesmontemagno/Xamarin.Plugins#101]
Hi
This is a big problem for us. It seems the package requires minimum api 21 (android 5.0). So something as simple like this:
var mediaFile = await CrossMedia.Current.PickVideoAsync();

will crash on your device if you compile or have anything else. That's more than 50% of the user base right now!

I understand sometimes things that require the new api are needed, but the Media package doesn't seem to require it at all for picking video. I actually got the source code and compiled it with Api 17 with no problem, and probably it can go to 14 even (android 4.0)
Could you lower the Api requirement please?

Thanks

**Comment by jamesmontemagno** [comment]
You do not need to set the minimum to API 21, you just need to set the Compile to to 21.

**Comment by joaquingrech** [comment]
I've tested it on an android 4.4... if I set it to target api 21 and minimum, api 14 for instance... I still got a crash

**Comment by jamesmontemagno** [comment]
My test settings are:
General -> Compile Framework: Android 5.1 API 22
Android Application -> Minimum Android Version Android 4.0.3 API 15
Android Application -> Target Android version Android 5.1 API 22

In Android Application (AndroidManifest) 
I set required permissions: WRITE_EXTERNAL_STORAGE & CAMERA permissions

Tested on Moto G with Android 4.4.4 on it. Working well here, didn't get a crash at all. Did you set all of these above?
